### PR TITLE
Cypress testing

### PR DESCRIPTION
**Pull request by EmilyClare4**
Cypress End-to-End testing

### CODE REVIEW

File: components/NavigationBar.vue
Comment (kloudsoft91) [inline components/NavigationBar.vue:1]: Hey, did you forget to add the data-cy parameter on stipeLen? I see you edited the line

File: cypress/e2e/spec.cy.ts
Comment (kloudsoft91) [inline cypress/e2e/spec.cy.ts:1]: Your comment says the final Url needs to be updated before merging, is this correct? It currently points to localhost

**Review by kloudsoft91 — DISMISSED** [review]
Looks good, conditionally approved (see comments on files). Will merge after conversations are resolved

File: .github/workflows/ci.yml
Comment (kloudsoft91) [inline .github/workflows/ci.yml:1]: Is there a step we could potentially add to do the Cypress testing in our workflow?

**Comment by kloudsoft91** [comment]
Apologies, one more thing, if there is a local cli command to do the e2e testing, please add necessary steps and information to the readme. e.g. change the url to visit in spec.cy.ts and then run command abc etc.

File: components/NavigationBar.vue
Comment (EmilyClare4) [reply components/NavigationBar.vue:1]: I see why you asked, but no, the filter works fine with just the stipeWidth parameter. Not sure why, we might need to add this as a bug/issue

File: cypress/e2e/spec.cy.ts
Comment (EmilyClare4) [reply cypress/e2e/spec.cy.ts:1]: Yes good catch. Deployment is currently failing on this branch so the tests won't work until merged with master

File: .github/workflows/ci.yml
Comment (EmilyClare4) [reply .github/workflows/ci.yml:1]: There is and I tried it, but it was failing because it couldn't find the package-lock.json file since we added it to .gitignore. If we add this back in perhaps I could get it to work just not sure on time

**Review by kloudsoft91 — APPROVED** [review]

**Pull request merged**
